pkg/deploy/generator/scripts/util-services.sh

```diff
@@ -286,17 +315,232 @@ ExecStart=/usr/bin/podman run \
   -v /run/systemd/journal:/run/systemd/journal \
   -v /var/etw:/var/etw:z \
   -v /var/run/mdsd/asa:/var/run/mdsd/asa:z \
-  $image \
+  $RPIMAGE \
```

Comment:
```suggestion
  ${RPIMAGE} \
```
Can we use some consistency with brace expansion in the same quote block?
Using braces is probably best, as it should evaluate to "" if it's unset (although I'm not aware of exactly how systemd handles unset variables).

Reply:
Thanks for highlighting this. If it is unset, system just keep it as it is and does not evaluate or expand it. So `${RPIMAGE}` remains `${RPIMAGE}`. For consistency, I have removed the curly braces for service files as that is the standard way